source/_integrations/sensor.avri.markdown

```diff
@@ -0,0 +1,52 @@
+---
+title: Avri Waste Sensor
```

Comment:
The title has to match the title in the integration `manifest.json`

Reply:
Done.